Sources/Yams/Node.Sequence.swift

```diff
@@ -146,24 +141,7 @@ extension Node.Sequence: MutableCollection {
     }
 }
 
-extension Node.Sequence: RandomAccessCollection {
-    // BidirectionalCollection
-    /// :nodoc:
-    public func index(before index: Index) -> Index {
-        return nodes.index(before: index)
-    }
-
-    // RandomAccessCollection
-    /// :nodoc:
-    public func index(_ index: Index, offsetBy num: Int) -> Index {
```

Comment:
It seems that only this func is not used in the code coverage before this PR. 
The other func to be deleted was used in the previous tests, so it seems to be no problem.